Packages/com.unity.inputsystem/InputSystem/Actions/Composites/AxisComposite.cs

```diff
@@ -37,33 +36,36 @@ namespace UnityEngine.InputSystem.Composites
     /// acceleration control(s), and setting <see cref="whichSideWins"/> to <see cref="WhichSideWins.Negative"/>,
     /// if the break button is pressed, it will always cause the acceleration button to be ignored.
     ///
-    /// The values returned are the actual actuation values of the buttons, unaltered for <see cref="positive"/>
-    /// and inverted for <see cref="negative"/>. This means that if the buttons are actual axes (e.g.
-    /// the triggers on gamepads), then the values correspond to how much the axis is actuated.
+    /// The actual <em>absolute</em> values of <see cref="negative"/> and <see cref="positive"/> are used
+    /// to scale <see cref="minValue"/> and <see cref="maxValue"/> respectively. So if, for example, <see cref="positive"/>
+    /// is bound to <see cref="Gamepad.rightTrigger"/> and the trigger is at a value of 0.5, then the resulting
+    /// value is <c>maxValue * 0.5</c> (the actual formula is <c>midPoint + (maxValue - midPoint) * positive</c>).
     /// </remarks>
     [DisplayStringFormat("{negative}/{positive}")]
     [DisplayName("Positive/Negative Binding")]
     public class AxisComposite : InputBindingComposite<float>
     {
         /// <summary>
-        /// Binding for the button that controls the positive direction of the axis.
+        /// Binding for the axis input that controls the negative [<see cref="minValue"/>..0] direction of the
+        /// combined axis.
         /// </summary>
         /// <remarks>
         /// This property is automatically assigned by the input system.
         /// </remarks>
         // ReSharper disable once MemberCanBePrivate.Global
         // ReSharper disable once FieldCanBeMadeReadOnly.Global
-        [InputControl(layout = "Button")] public int negative = 0;
+        [InputControl(layout = "Axis")] public int negative = 0;
```

Comment:
The xml docs for AxisComposite still refer to the negative and positive controls as buttons.

Reply:
Fixed